testsuite/forge-cli/src/main.rs

```diff
@@ -1693,53 +1697,54 @@ fn realistic_env_max_load_test(
 }
 
 fn realistic_network_tuned_for_throughput_test() -> ForgeConfig {
```

Comment:
How about adding a new test and leave the current one unchanged?

Reply:
The original runs at 6k iirc, I don't think it's useful or establish any baseline?

Reply:
Or I'm missing how it's used currently? You and @igor-aptos  track it anywhere?

Reply:
I thought the original runs at ~8k TPS and is included in the list of stable forge test that runs daily.. @igor-aptos can confirm..

Reply:
Good catch then. Gonna update.

Reply:
Or maybe just comment out the resource ask out and update the numbers in CI? ( I have difficulty naming things. :trollface: 
 )